ext/curl/curl.stub.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/** @param resource $handle */
+function curl_close($handle) : void {}
```

Comment:
For consistency with other stubs, there should not be a space between the closing parenthesis of a function and its return type colon. It should be written as `function curl_close($handle): void {}`. Same with all the other functions.